torchaudio/functional.py

```diff
@@ -118,15 +119,15 @@ def istft(
     """
     stft_matrix_dim = stft_matrix.dim()
     assert 3 <= stft_matrix_dim, "Incorrect stft dimension: %d" % (stft_matrix_dim)
-    assert stft_matrix.nelement() > 0
+    assert stft_matrix.numel() > 0
```

Comment:
STFT is decorated with `@torch.jit.ignore` and so is not compiled. What are the changes to STFT?

Reply:
This is also a JITability workaround; `nelements` and `numel` are basically the same functions but one is supported by Torchscript and the other is not.

Reply:
Opened issue pytorch/pytorch#31626 for this.